lib/flake.nix

```
@@ -1,5 +1,10 @@
 {
   description = "Library of low-level helper functions for nix expressions.";
 
-  outputs = { self }: { lib = import ./.; };
+  outputs = { self }:
+    let
+      lib' = import ./.;
+    in {
+      lib = lib'.extend (import ./__flake-version-info.nix self);
```

`'` is often used for the second state rather than the first, particularly in math.

```suggestion
      lib0 = import ./.;
    in {
      lib = lib0.extend (import ./__flake-version-info.nix self);
```

```suggestion
      libBase = import ./.;
    in {
      lib = libBase.extend (import ./__flake-version-info.nix self);
```